Sources/EasyImagy/ImageProtocol.swift

```diff
@@ -96,6 +97,7 @@ extension ImageProtocol {
 extension ImageProtocol {
     public func transposed() -> Image<Pixel> {
         var pixels = [Pixel]()
+        pixels.reserveCapacity(xRange.count * yRange.count)
```

Comment:
`xRange.count * yRange.count` can be replaced with `count` because `ImageProtocol` has the `count` property. Overhead to call `xRange.count` and `yRange.count` indirectly can be expected to be removed by inline expansion when they are built with optimization.